src/home/Home.jsx

```diff
@@ -1,3 +1,256 @@
+            </div>
+            {topAVS.map((entry, i) => (
+              <div
+                key={`top-avs-item-${i}`}
+                className={`border-t border-outline flex flex-row gap-x-2 justify-between items-center p-4 hover:bg-default`}
+              >
+                <img src={entry.avs.logo} className="size-5 rounded-full" />
+                <span className="basis-full truncate">{entry.avs.name}</span>
+                <span className="basis-2/3">
+                  {formatNumber(entry.operators, compact)}%
+                </span>
+                <span className="basis-1/2 text-end">
+                  <div>{formatNumber(entry.tvl.eth, compact)} ETH</div>
+                  <div className="text-foreground-1 text-xs">
+                    $ {formatNumber(entry.tvl.usd, compact)}
+                  </div>
+                </span>
+              </div>
+            ))}
+          </div>
+        </Card>
+        <Card
+          radius="md"
+          className="bg-content1 border border-outline space-y-4 p-4 w-full"
+        >
+          <div className="font-light text-base text-foreground-1">
+            Top Operators
+          </div>
+          <div className="text-sm">
+            <div className="flex flex-row gap-x-2 justify-between items-center p-4 text-foreground-1">
+              <span className="basis-full">Operators</span>
+              <span className="basis-2/3">Restakers</span>
+              <span className="basis-1/2 text-end">TVL</span>
+            </div>
+            {topOperators.map((entry, i) => (
+              <div
+                key={`top-avs-item-${i}`}
+                className={`border-t border-outline flex flex-row gap-x-2 justify-between items-center p-4 hover:bg-default`}
+              >
+                <img
+                  src={entry.operator.logo}
+                  className="size-5 rounded-full"
+                />
+                <span className="basis-full truncate">
+                  {entry.operator.name}
+                </span>
+                <span className="basis-2/3">
+                  {formatNumber(entry.restakers, compact)}%
+                </span>
+                <span className="basis-1/2 text-end">
+                  <div>{formatNumber(entry.tvl.eth, compact)} ETH</div>
+                  <div className="text-foreground-1 text-xs">
+                    $ {formatNumber(entry.tvl.usd, compact)}
+                  </div>
+                </span>
+              </div>
+            ))}
+          </div>
+        </Card>
+      </div>
+
+      <OverviewTVLOverTime width={1100} height={400} />
+      <OverviewLRTDistribution />
+    </div>
+  );
+}
+
+const CTAs = () => {
+  return (
+    <div className="space-y-2">
+      <Card
+        radius="md"
+        className="bg-cinder-1 hover:bg-cinder-1/80 cursor-pointer flex flex-row items-center justify-normal gap-3 px-4 py-2.5"
+      >
+        <div>
+          <img src="/public/nethermind.png" />
```

Comment:
It was always be relative to the configured static asset dir
```suggestion
          <img src="/ethermind.png" />
```